docs/src/prototypes/chatMessages/ControlMessages/ControlMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { Ref, ChatMessageProps, Chat, Icon } from '@stardust-ui/react'
+
+interface ControlMessageProps {
+  focused?: boolean
+  icon?: boolean
+  message: ChatMessageProps
+}
+class ControlMessage extends React.Component<ControlMessageProps> {
+  messageRef = React.createRef<HTMLElement>()
+
+  componentDidMount() {
+    if (this.props.focused && this.messageRef) {
+      this.messageRef.current.focus()
+    }
+  }
+
+  render() {
+    return (
+      <>
+        {this.props.icon ? <Icon name="participant-add" /> : null}
+        <Ref innerRef={this.messageRef}>
+          <Chat.Message
+            {...this.props.message}
+            styles={{
+              padding: 0,
+              marginLeft: '10px',
+              backgroundColor: '#f3f2f1',
```

Comment:
Can we have Provider around this prototype, and defined overrides there? That's way it would be more obvious what is happening if we replace this color with `colorScheme.default.background2` for example

Reply:
I was thinking about this. Will do